Samples/tutorials/first-run-experience-tutorial/My Office Add-in/src/commands/commands.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+ * See LICENSE in the project root for license information.
+ */
+
+/* global Office */
+
+Office.onReady(() => {
+  // If needed, Office.js is ready to be called.
+});
+
+/**
+ * Shows a notification when the add-in command is executed.
+ * @param event {Office.AddinCommands.Event}
+ */
+function action(event) {
+  const message = {
+    type: Office.MailboxEnums.ItemNotificationMessageType.InformationalMessage,
```

Comment:
We should consider removing this Outlook-specific content from the completed tutorial. I know it's included from yo office, but the user doesn't touch this in the tutorial and it's confusing (to me) to include it.